ui/src/components/erc/Inspect/DataView/Download/Download.js

```diff
@@ -1,28 +1,28 @@
 import React from 'react';
 
-import config from '../../../../../helpers/config'
+//import config from '../../../../../helpers/config'
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.